tox.ini

```diff
@@ -46,7 +46,7 @@ deps =
 	-rrequirements/tests.txt
 commands =
     pip install -e .
-	behave --tags=-disabled integration_tests {posargs}
+	behave --tags=-disabled --tags=-slow integration_tests {posargs}
```

Comment:
I hope that non of the tests tagged with `slow` failed?

Reply:
@hannesdiedrich nope, I just added this to improve the runtime of the tests, reverted it now